### PR TITLE
CLDR-14202 B Simple tool for checking conversions

### PR DESCRIPTION
**Pull request by macchiati**
##### Checklist

- [ ] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-14202
- [ ] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

**Comment by macchiati** [comment]
Changing fixVersion to 39, since the code for the tool doesn’t have to be in 38. However, I'd like to go ahead and get it merged. We can enhance further in 39.

File: tools/java/org/unicode/cldr/tool/VerifyConverterResults.java [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+        }
+    }
+
+    /*
+    public static void main(String[] args) throws UnsupportedEncodingException {
+        for (final String baseName : getBaseNames()) {
+            gatherData(baseName);
+        }
+    }
+    // Ugly hack to get base names
```
Comment (srl295) [inline tools/java/org/unicode/cldr/tool/VerifyConverterResults.java:289]: there's not a great way to do this

**Comment by macchiati** [comment]
Yes I know that it's not the right way of reading the resource bundles but I ran out of time to do it. Wanted to focus first on the Jason side. Need to work on it more in 39.

**Comment by srl295** [comment]
> Yes I know that it's not the right way of reading the resource bundles but I ran out of time to do it. Wanted to focus first on the Jason side. Need to work on it more in 39.

Right, what I meant is that I am not convinced there *is* a clean way to do it from where this code sits.  One idea might be to actually read the LDML2ICUConverter config data, however, that makes the verification a little more circular and less clean-slate.

**Comment by macchiati** [comment]
I. Think we can run the tool after integrations, and just examine resource
bundles.

**Pull request merged**